workflow/controller/operator.go

```diff
@@ -1290,7 +1295,11 @@ func (woc *wfOperationCtx) createPVCs() error {
 				},
 			},
 		}
-		woc.wf.Status.PersistentVolumeClaims[i] = vol
+		woc.wf.Status.PersistentVolumeClaims = append(woc.wf.Status.PersistentVolumeClaims, vol)
+		// Clearing previous PVC forbidden error
+		if woc.wf.Status.Message != "" && strings.Contains(woc.wf.Status.Message, "Waiting for a PVC to be created") {
```

Comment:
minor - extract "Waiting for a PVC to be created" to a const and use on line 288